ui/raidboss/data/05-shb/raid/e10s.js

```diff
@@ -244,9 +241,84 @@ export default {
           cn: '影子放到外圈',
         },
       },
+      run: (data) => data.clones = true,
+    },
+    {
+      // This checks your shadow's job against your job, since your shadow has
+      // the same job as you. If there's multiple of one job, or a shadow has
+      // a job of 0 (player died), then return '?' for the affected players.
+      id: 'E10S Shadow Of A Hero',
+      netRegex: NetRegexes.addedCombatantFull({ name: 'Shadow Of A Hero' }),
+      condition: (data) => data.clones,
+      run: (data, matches) => {
+        data.myClone = data.myClone || [];
+        const clonesJob = parseInt(matches.job, 16).toString();
+        if (clonesJob === Util.jobToJobEnum(data.job))
+          data.myClone.push(matches.id.toUpperCase());
+      },
+    },
+    {
+      id: 'E10S Shadow Of A Hero Head Marker Map',
+      netRegex: NetRegexes.headMarker({ target: 'Shadow Of A Hero' }),
+      condition: (data) => !data.shadowMarkerMap,
+      suppressSeconds: 1,
+      run: (data, matches) => {
+        data.shadowMarkerMap = {};
+        const idPivot = parseInt(matches.id, 16);
+        for (let i = 0; i < 3; ++i) {
+          const hexPivot = (idPivot + i).toString(16).toUpperCase().padStart(4, '0');
+          data.shadowMarkerMap[hexPivot] = i + 1;
+        }
+      },
+    },
+    {
+      id: 'E10S Shadow Of A Hero Head Marker',
+      netRegex: NetRegexes.headMarker({ target: 'Shadow Of A Hero' }),
+      condition: (data) => !data.headMarkerTriggered,
+      durationSeconds: 7,
+      alertText: (data, matches, output) => {
+        if (!data.myClone || data.myClone.length !== 1) {
+          data.headMarkerTriggered = true;
+          return output.unknown();
+        }
+        if (matches.targetId === data.myClone[0]) {
+          data.headMarkerTriggered = true;
+          return output[data.shadowMarkerMap[matches.id]]();
+        }
+      },
+      outputStrings: {
+        '1': {
+          en: '1',
+          de: '1',
+          fr: '1',
+          cn: '1',
+          ko: '1',
+        },
+        '2': {
+          en: '2',
+          de: '2',
+          fr: '2',
+          cn: '2',
+          ko: '2',
+        },
+        '3': {
+          en: '3',
+          de: '3',
+          fr: '3',
+          cn: '3',
+          ko: '3',
+        },
+        'unknown': {
+          en: '?',
+          de: '?',
+          fr: '?',
+          ja: '?',
+          cn: '?',
+          ko: '?',
+        },
+      },
```

Comment:
would you like to add JA translation as well?
which is the same as others, just simply "1", "2" and "3".